src/main/java/org/opensearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -419,7 +419,7 @@ public void logDocumentWritten(ShardId shardId, GetResult originalResult, Index
                     try (XContentParser parser = XContentHelper.createParser(NamedXContentRegistry.EMPTY, THROW_UNSUPPORTED_OPERATION, originalResult.internalSourceRef(), XContentType.JSON)) {
                         Object base64 = parser.map().values().iterator().next();
                         if(base64 instanceof String) {
-                            originalSource = (new String(BaseEncoding.base64().decode((String) base64)));
+                            originalSource = (new String(BaseEncoding.base64().decode((String) base64), StandardCharsets.UTF_8));
```

Comment:
Since we use this in numerous places and I believe there used to be a content type setting in a separate file, do we want to make a setting for the encoding method or does hard coding make more sense because we are setting it to the most broad option?

Reply:
When we were on unix environments the setting value was always UTF-8 both inside of security and OpenSearch so everything was happy.  Since OpenSearch widely forces the use of UTF-8 and security does in some spots this was the source of our issues.